agent/acs/model/api/api-2.json

```diff
@@ -455,6 +455,18 @@
       "type":"list",
       "member":{"shape":"PortMapping"}
     },
+    "ProxyConfiguration":{
+      "type":"structure",
+      "members":{
+        "type":{"shape":"ProxyConfigurationType"},
+        "containerName":{"shape":"String"},
+        "properties":{"shape":"StringMap"}
+      }
```

Comment:
Please only includes changes that related to ur feature. Normally we only update changes related to our own features when we update acs model.